locale/fr/locale.cfg

```
@@ -14,15 +14,18 @@ add_top=[Add to Top]
 persist=Enregistrer
 cancel=Annuler
 update=MAJ
-delete=Delete
-confirm_deletion=Confirm deletion
+delete=Supprimer
+confirm_deletion=Confirmer la suppression
+
+import=import
+export=export
```

```suggestion
export=exporter
```
This one, as the one above may have to be corrected, I have to check the whole string and sentences where it is used. Should be correct if only used as a label.

---

So far it is only used as label :) Thanks